all-services.jenkinsfile

```diff
@@ -0,0 +1,158 @@
+pipeline
+{
+    agent any
+    environment
+    {
+      RELEASE_VERSION = "JENKINSTEST"
```

Comment:
@cookersjs How do you get the current release version in your Jenkins files?

Reply:
It is based on the current working directory: https://github.com/reactome/release-orthoinference/blob/develop/Jenkinsfile#L14

Reply:
Should I just copy that line to here? Or is there anything else?

Reply:
Also, would there be a way to do this that is less dependent on the file system?

Reply:
You could do it via a secret potentially. This would just involve you specifying the release number each time as a secret/credential. I looked into doing it this way but using the file system was a good solution for automatically detecting the release number, which made it so that only the master config file required release-specific variables

Reply:
Jenkins creates the Release## directory, right? So if you run a full release from start to finish from Jenkins, you shouldn't have to worry about the directory having the correct name, is that right?

Reply:
No, the user has to create the new directory named '##' by copying it from the Release_Template folder. Basically the sequence is:
1) Create new ## folder that is copied from 'Release_Template'
2) Upload master configuration file as a locally scoped credential in the ## folder

You can check out slightly out-of-date documentation here: https://docs.google.com/document/d/1JaSaqbxPu1FVIOcM0h5YJpA0EQKyBcD1kY81boLBWKs/edit?usp=sharing

For what you are asking about it should all still be valid

Reply:
But that "folder" they create is a Jenkins object - they don't have to go into the filesystem and `mkdir Release$NUMBER` or something like that.

Reply:
Thats right, its created via the Jenkins GUI

Reply:
Ok. I'll make the changes to get the release number from the current directory. Do you want to do your own test of this new pipeline on the Release server?

Reply:
I'd love to, but I am going to be running a mock release shortly and I'm not sure about testing this while that is going on. Maybe the window between the mock release and actual release?